File: 2_condition_and_loop/check_triange.py
# Write a program to check a triange is valid or not


def check_triange(sideA, sideB, sideC):
    if (sideA > sideB+sideC) or (sideB > sideA+sideC) or (sideC > sideA+sideB):
        print("No, the lengths wont form a triange")
    elif(sideA == sideB+sideC) or (sideB == sideA+sideC) or (sideC == sideA+sideB):
        print("Yes, it can form a degenerated triangle")
    else:
        print("Yes, a triangle can be formed out of it")


sideA = int(input("digit the sideA: "))
sideB = int(input("digit the sideB: "))
sideC = int(input("digit the sideC: "))
check_triange(sideA, sideB, sideC)

File: ruby_and_python/2_condition_and_loop/check_triange.rb
def check_triange(sideA, sideB, sideC)
  if (sideA > sideB+sideC) || (sideB > sideA+sideC) || (sideC > sideA+sideB)
    puts "No, the lengths wont form a triangle"
  elsif (sideA == sideB+sideC) || (sideB == sideA+sideC) || (sideC == sideA+sideB)
    puts "Yes, it can form a degenerated triangle"
  else
    puts "Yes, a trinagle can be formed out of it"
  end
  
end


print "sideA value: "
sideA = gets.chomp.to_i
print "sideB value: "
sideB = gets.chomp.to_i
print "sideC value: "
sideC = gets.chomp.to_i

check_triange(sideA, sideB, sideC)

